### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by sdogruyol**
The time has come to finally make Kemal 1.0.0 after 5+ years 🚀 

All the specs are already green in Crystal 1.0.0-dev 👍 

Waiting on https://github.com/crystal-lang/crystal/pull/10500 😄

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ development_dependencies:
     github: crystal-ameba/ameba
     version: ~> 0.13.4
 
-crystal: 0.35.0
+crystal: 1.0.0
```
Comment (straight-shoota) [inline shard.yml:23]: Maybe this should include `0.35.0` if it's still downward compatible?

Comment (sdogruyol) [reply shard.yml:23]: Good point, actually it's compatible with `0.36.x`so `>= 0.36.0` is fine 👍

File: shard.yml [hunk 2]
```diff
@@ -20,6 +20,6 @@ development_dependencies:
     github: crystal-ameba/ameba
     version: ~> 0.13.4
 
-crystal: 0.35.0
+crystal: => 0.36.0
```
Comment (Sija) [inline shard.yml:23]:
```suggestion
crystal: ">= 0.36.0"
```

Comment (sdogruyol) [reply shard.yml]: Thanks

**Comment by waghanza** [comment]
Compilation seems to be broken @sdogruyol 
https://the-benchmarker.semaphoreci.com/jobs/69b4388c-871a-4fd3-b289-1eae3ece8a3c

**Comment by sdogruyol** [comment]
@waghanza radix and kilt needs to be updated to 1.0.0

**Comment by waghanza** [comment]
but need a further release no ? (in case the release from their side is a minor, not a patch)

btw, I'll track `radix` and `kilt`

**Comment by sdogruyol** [comment]
I'm not sure why they fail also..they don't specify a spefici Crystal version in there `shards.yml` @jeromegn @luislavena

**Comment by waghanza** [comment]
shards resolver I suppose https://crystal-lang.org/2020/08/06/shards-0.12.0-released.html

**Comment by sdogruyol** [comment]
found it https://github.com/crystal-loot/exception_page/issues/20

**Comment by waghanza** [comment]
ℹ️  https://github.com/luislavena/radix/pull/32

**Pull request merged**